src/View/StringTemplate.php

```diff
@@ -284,7 +306,7 @@ protected function _formatAttribute($key, $value, $escape = true)
         $truthy = [1, '1', true, 'true', $key];
         $isMinimized = isset($this->_compactAttributes[$key]);
         if ($isMinimized && in_array($value, $truthy, true)) {
-            return "$key=\"$key\"";
+            return "$key";
```

Comment:
Well, I don't mind either way :) Just saying that you will need to fix a lot of more tests for the short version.